application/src/main/java/org/togetherjava/tjbot/features/moderation/history/PurgeMessageListener.java

```diff
@@ -58,7 +58,7 @@ private void updateHistory(MessageReceivedEvent event) {
 
             incrementRecordsCounter();
         } else {
-            logger.warn("purge history reached limit");
+            logger.debug("purge history reached limit");
```

Comment:
the log would be more useful if it also gives a suggestion how to fix the problem.

"Purge History reached limit. Either delete some rows of the table or wait for them to expire."

Reply:
Actually i think i might have a better idea, Imagine the DB is maxed out with 7.5k messages. No more records are being saved, which means any new spam messages would be ignored and using `purge-in-channel` would only delete older messages which might not be the best thing . Ideally i think we should keep the messages from newest to oldest upto the "max limit". 

Possible Solution:
Let's say we have a limit 7.5k, we hit a mark of 7.4k. A routine should be trigged that only clears out last 500 records or so. This way all the new messages are being tracked and command is not useless past the limit. Thoughts ?